# Assignment 6

To measure the psychological state of Americans, a psychology researcher (in 1929) devised
three questions put to 3443 Americans from different walks of life. He asked them whether they
were happy. They could answer only **yes** or **no** to each question. By today’s standards the study
is quite simple, and he did no formal statistical analysis of his data. The frequency data (number
of **yes** and **no** responses within each employment category) from this study are in the file
*Americandata.csv*.

---

You need to use the data and answer the question, that is, **Is the type of profession related to the happiness of American people?**
Report the **χ2** statistic and the *statistical significance of test*, in addition to the contingency table.
Also, explain the results and what they mean. What does the value of **χ2** statistic mean? Is it
good or bad? Is the test statistically significant?

In [ ]:
#code here
import pandas as pd
from scipy.stats import chi2_contingency

df = pd.read_csv("./Americandata.csv")

contingency_table = df.set_index("Happy")
print("The Contingency Table:")
print(contingency_table)

chi2, p, dof, expected = chi2_contingency(contingency_table)

print("\nChi-square statistic:", chi2)
print("p-value:", p)
print("Degrees of freedom:", dof)

The Contingency Table:
       Students  Laborers  Preachers  Physicians  Housewives  Teachers  \
Happy                                                                    
Yes         390       378         35         159          78       108   
No         1610       122        265          51         122        38   

       Lawyers  Musicians  
Happy                      
Yes         11         31  
No          64         19  

Chi-square statistic: 936.1394782304601
p-value: 7.523435335099551e-198
Degrees of freedom: 7


Put your analysis in the empty `markdown` cell below.

The Chi-square statistic (χ2) is: 936.1394782304601 and the p-value (statistical significance of test): 7.523435335099551e-198.

The very large χ2 value is very differnt then what we would see if they weren't related. Also, this isn't good or bad it just shows a big difference in the groups.

The p-value is very small and under 0.05 so profession and happiness were shown to be related.